src/variants/ubuntu_arm64/post_chroot_script

```diff
@@ -15,3 +15,9 @@ apt-get update
 apt-get -y --force-yes install libraspberrypi-bin rpi.gpio-common
 apt-get clean
 apt-get autoremove -y
+
+# Compatibility for ws281x
+sudo ln config-5.8.0-1006-raspi config.txt
+sudo touch cmdline.txt
```

Comment:
In the next release of the plugin, the UI does not crash if these files aren't present. Can be found on the development branch currently. Had the same issue with running it in docker, I don't know if it is a good idea to create empty files that don't do anything, since the plugin/user might think that everything is OK but actually it is not.